src/iminuit/util.py

```diff
@@ -874,7 +874,12 @@ def _arguments_from_docstring(obj):
     #   min(arg1, arg2, *args, *[, key=func]) -> value
     #   Foo.bar(self, int ncall_me =10000, [resume=True, int nsplit=1])
 
-    name = obj.__name__
+    try:
+        name = obj.__name__
+    except AttributeError:
+        # See Issue #608
+        # https://github.com/scikit-hep/iminuit/issues/608
+        return ()
```

Comment:
I think that is fine for now, but I wonder whether it is possible to get the function name in a more generic way, perhaps with the inspect module so that it also works with a wrapped function.

Reply:
It is! Partial objects have an attribute `partial.func`. This attribute basically holds the function object itself.

However, once you go down this rabbit hole, you will eventually end up writing a completely new function just for partial objects. Keep in mind that `_arguments_from_inspect` **already works perfectly fine for partial objects**. The only problem is that `_arguments_from_inspect` returns `()` (and rightfully so) because it only finds `*args` which it ignores. Thus, we execute the next function in the chain which is `_arguments_from_docstring` which simply does not work for partial objects.

Reply:
Ok, let's keep it like this for now.